beacon-light-client/circom/scripts/common.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+look_for_ptau_file() {
+  if [ -f "$PHASE1" ]; then
+    echo "Found Phase 1 ptau file"
+  else
+    echo "No Phase 1 ptau file found. Exiting..."
+    exit 1
+  fi
+}
```

Comment:
Make function takes the path as an argument.
```suggestion
look_for_ptau_file() {
  local phase1_file="$1"
  if [ -f "$phase1_file" ]; then
    echo "Found Phase 1 ptau file"
  else
    echo "No Phase 1 ptau file found. Exiting..."
    exit 1
  fi
}
```
Then use it like that
```sh
look_for_ptau_file "$PHASE1"
```

This ensures the function remains flexible with different environment variable names